pycbuf/CBufParserPy.cpp

```diff
@@ -457,14 +453,16 @@ PyTypeObject* CBufParserPy::GetPyTypeFromCBuf(uint64_t hash, ast_struct* st, PyO
   PyTypeObject* pyType = nullptr;
   if (state->info_map->contains(hash)) {
     pyType = (PyTypeObject*)(*state->info_map)[hash].type;
-    VLOG_ASSERT(pyType != nullptr);
-    VLOG_ASSERT(PyType_Check(pyType) != 0);
+    if (pyType == nullptr) return nullptr;
```

Comment:
would it make sense to have a cassert macro to be able to have more introspection when there are errors? I have spent a long time trying to figure out who set a PyObject to null